tests/gold_tests/timeout/tls_conn_timeout.test.py

```diff
@@ -63,7 +63,7 @@
 # Request the port that should timeout in the handshake
 # Should retry once
 tr = Test.AddTestRun("tr-blocking-post")
-tr.Setup.Copy("../chunked_encoding/ssl/server.pem")
+tr.Setup.Copy("../../tools/ssl/server.pem")
```

Comment:
Instead of ../../tools, I believe we can use the `Test.Variables.AtsTestToolsDir`.